packages/loader/container-loader/src/naming.md

```diff
@@ -10,19 +10,21 @@
       * Inbound/Outbound are operational (...is there a possible race condition if `IContainerLoadMode.opsBeforeReturn === undefined`?)
```

Comment:
@vladsud - is there a possible race condition if `IContainerLoadMode.opsBeforeReturn === undefined` where `connected` event could fire but inbound queue is not operational yet?  Does this matter in terms of guarantees we make about what this event means?

Reply:
I'm not sure what this comment means. Only controls when container is returned, it has nothing to do with controlling connections on boot (which is controlled by IContainerLoadMode.deltaConnection)
I'd remove this comment.

Reply:
I'll try to explain again in our 1-1 this afternoon